sphinx/writers/html.py

```diff
@@ -429,6 +429,11 @@ def depart_title(self, node: Element) -> None:
 
         super().depart_title(node)
 
+    # overwritten
+    def depart_rubric(self, node: Element) -> None:
+        self.body.append('</p>\n')
+        self.add_permalink_ref(node.parent, _('Permalink to this headline'))
```

Comment:
What is a parent node of the rubric node? I think this is a wrong implementation.

Reply:
Usually a parent node for `rubric` is a section, or anything with an `ids`.

Reply:
That's strange. The section and rubric node is not related at all. I guess the anchor generated `add_permalink_ref` will link to the section instead of rubric itself. It's strange.

Reply:
Ah, I understand why you specified the section node here. It seems your extension creates a rubric node just under the section node. But it is prohibited. The section node should start with a title node.

>section elements begin with a title, and may contain body elements as well as transition, topic, and sidebar elements.
>https://docutils.sourceforge.io/docs/ref/doctree.html#section